fs_image/bzl/oss_shim_impl.bzl

```diff
@@ -148,8 +154,8 @@ def _python_library(
     python_library(
         name = name,
         deps = _normalize_deps(deps),
-        resources = _invert_dict(resources),
-        srcs = _invert_dict(srcs),
+        resources = _invert_and_normalize_dict(resources),
+        srcs = _invert_and_normalize_dict(srcs),
```

Comment:
I think we should leave this on just as `_invert_dict(srcs)`.  see my comment above about the `_normalize_resources` method.  The reason I think we should leave this as `_invert_dict` is for two reasons:

1, this is really only needed for one case and that is the generation of the `python_library` inside `image_unittest_helpers.bzl` that wraps the invocation of `nspawn`.  And 2, because I don't think we should support stripping out `facebook/` paths in `srcs` if we use them, that could potentially be a source of bugs and I just don't think we should support it. If there's something entirely internal it should be a separate top level target in a `facebook/` dir, not a src.